### PR TITLE
Feature/data writer

### PR DESCRIPTION
**Pull request by vancouver29**
each line in the file will look like this : (mass, position(x, y), velocity(dx, dy))

### CODE REVIEW

File: src/util/mod.rs [hunk 1]
```diff
@@ -30,18 +32,23 @@ impl DataWriter {
 
     /// Creates a new file in the writers directory with each point written
     /// on a separate line.
-    pub fn write(&mut self, points: Vec<Point>) {
```
Comment (johnxnguyen) [inline src/util/mod.rs:33]: Why are you changing the existing method? I thought you were going to define a new method to be able to save list of bodies?

Comment (vancouver29) [reply src/util/mod.rs:33]: I changed it from points: Vec<Point> to bodies: Vec<Body>

Comment (johnxnguyen) [reply src/util/mod.rs:33]: Yes I see that, but why did you change it?

Comment (vancouver29) [reply src/util/mod.rs:33]: I thought if the concept for bodies is similar to points, so I changed it ^^! so I cant use your already defined function ?

Comment (johnxnguyen) [reply src/util/mod.rs:33]: But we still need the method to write only the points. If you change it, we no longer have this method anymore.

Comment (vancouver29) [reply src/util/mod.rs:33]: ah right, I see now

**Pull request closed without merging**